tools/check_format.py

```diff
@@ -75,6 +75,9 @@
     "ProtobufWkt::MapPair":             "Protobuf::MapPair",
     "ProtobufUtil::MessageDifferencer": "Protobuf::util::MessageDifferencer"
 }
+CPP_STD_ERRORS = {
+    "absl::make_unique": "std::make_unique",
```

Comment:
do you want to put the "<" into both the 'from' and 'to' mappings here to make it less likely to mess with comments?

Reply:
Good point! Fixed